app/src/main/java/com/github/geohunt/app/model/database/api/Claim.kt

```diff
@@ -18,15 +19,22 @@ interface Claim {
     val challenge: LazyRef<Challenge>
 
     /**
-     * Unique identifier of the user that claimed the corresponding challenge
+     * Reference to the image of the claimed image
      */
-    val user: String
+    val image: LazyRef<Bitmap>
```

Comment:
So I don't think we have discussed adding the image clicked to the database yet based on the existing api. Would it be possible to remove this since a lot of the upcoming code from another PR doesn't take this into account as it wasn't discussed earlier and it might break a lot of things?

Reply:
This is not a change that I have done. I don't think it is a good idea to remove the code that someone else has written, as it could lead to conflicts in the future. Instead, this line of code can remain for now where it is, and the question whether it will stay for longer or not should depend on the code of challenge-view when it gets merged onto main.